santis/packages.yaml

```diff
@@ -14,3 +14,8 @@ packages:
     externals:
     - spec: libfabric@1.15.2.0
       prefix: /opt/cray/libfabric/1.15.2.0/
+  slurm:
+    buildable: false
+    externals:
+    - spec: slurm@23.02.6
```

Comment:
I just noticed that the other system configs have the version in format `x-y-z`, and so do the versions in the spack package. `23.02.6` is what `spack external find` filled in for me, but perhaps better to change to `23-02-6`?

Reply:
It is better to go with `23-02-6` for SLURM - that is how SLURM presents versions.

Reply:
Changed to `23-02-6`.